Xamarin.RangeSlider.Forms.Droid/RangeSliderRenderer.cs

```diff
@@ -64,68 +66,87 @@ private void UpdateControl(RangeSliderControl control, RangeSlider element)
         {
             base.OnElementPropertyChanged(sender, e);
-            switch (e.PropertyName)
-            {
-                case RangeSlider.LowerValuePropertyName:
-                    Control.SetSelectedMinValue(Element.LowerValue);
-                    break;
-                case RangeSlider.UpperValuePropertyName:
-                    Control.SetSelectedMaxValue(Element.UpperValue);
-                    break;
-                case RangeSlider.MinimumValuePropertyName:
-                case RangeSlider.MaximumValuePropertyName:
-                    Control.SetRangeValues(Element.MinimumValue, Element.MaximumValue);
-                    break;
-                case RangeSlider.MaxThumbHiddenPropertyName:
-                    Control.MaxThumbHidden = Element.MaxThumbHidden;
-                    break;
-                case RangeSlider.MinThumbHiddenPropertyName:
-                    Control.MinThumbHidden = Element.MinThumbHidden;
-                    break;
-                case RangeSlider.StepValuePropertyName:
-                    Control.StepValue = Element.StepValue;
-                    break;
-                case RangeSlider.StepValueContinuouslyPropertyName:
-                    Control.StepValueContinuously = Element.StepValueContinuously;
-                    break;
-                case RangeSlider.BarHeightPropertyName:
-                    if (Element.BarHeight.HasValue)
-                        Control.SetBarHeight(Element.BarHeight.Value);
-                    break;
-                case RangeSlider.ShowTextAboveThumbsPropertyName:
-                    Control.ShowTextAboveThumbs = Element.ShowTextAboveThumbs;
-                    ForceFormsLayout();
-                    break;
-                case RangeSlider.TextSizePropertyName:
-                    Control.TextSizeInSp = (int)Font.SystemFontOfSize(Element.TextSize).ToScaledPixel();
-                    ForceFormsLayout();
-                    break;
-                case RangeSlider.TextFormatPropertyName:
-                    Control.TextFormat = Element.TextFormat;
-                    break;
-                case RangeSlider.TextColorPropertyName:
-                    if (Element.TextColor != Xamarin.Forms.Color.Default)
-                        Control.TextAboveThumbsColor = Element.TextColor.ToAndroid();
-                    break;
-                case RangeSlider.FormatLabelPropertyName:
-                    Control.FormatLabel = Element.FormatLabel;
-                    break;
-                case RangeSlider.ActiveColorPropertyName:
-                    if (Element.ActiveColor != Xamarin.Forms.Color.Default)
-                        Control.ActiveColor = Element.ActiveColor.ToAndroid();
-                    break;
-                case RangeSlider.MaterialUiPropertyName:
-                    Control.MaterialUI = Element.MaterialUI;
-                    break;
+            if (e.PropertyName == RangeSlider.LowerValueProperty.PropertyName)
```

Comment:
Please use switch.

Reply:
Switch statements only work for constants. This is part of the change I mentioned above around removing the level of indirection as the more direct way to check which property has been changed is by looking at something like RangeSlider.LowerValueProperty.PropertyName. If you'd like to keep your constants and switch statements then you can revert the commits 8b0b9742dc7d444b26873fce3f459c6c78cf690b and 148507478e72de78afe0c10fd38bfca2b18f8d98